Model/Sales/TrackTraceHolder.php

```diff
@@ -195,6 +194,24 @@ public function convertDataFromMagentoToApi($magentoTrack, $options)
         return $this;
     }
 
+    /**
+     * @param $checkoutData
+     *
+     * @return string
+     */
+    public function getDeliveryDate(string $checkoutData): string
```

Comment:
Misschien moet de benaming van de method anders.
Als ik dit nu lees verwacht je alleen de delivery date terug, terwijl je veel meer terug krijgt.

Reply:
Misschien kan je ook in een command zetten wat de code precies doet, zodat het duidelijker is waarom je de huidige datum pakt +1 dag

Reply:
done